mods/tuxemon/maps/maple_house.tmx

```diff
@@ -59,16 +61,36 @@
     <property name="cond2" value="not npc_exists npc_wife"/>
    </properties>
   </object>
-  <object id="39" name="nodrinkpls" type="event" x="96" y="48" width="32" height="16">
+  <object id="39" name="nodrinkpls" type="event" x="96" y="32" width="32" height="16">
    <properties>
     <property name="act1" value="npc_face npc_wife, up"/>
     <property name="act2" value="translated_dialog nodrink"/>
     <property name="act3" value="npc_face npc_wife,down"/>
-    <property name="cond1" value="is player_at"/>
-    <property name="cond2" value="is player_facing up"/>
+    <property name="cond1" value="is player_facing_tile"/>
     <property name="cond3" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
+  <object id="40" name="create knight" type="event" x="192" y="96" width="16" height="16">
+   <properties>
+    <property name="act1" value="create_npc knight3,7,11,knight,stand"/>
```

Comment:
Remove the sprite name from this command, that functionality is deprecated and will generate warnings in the logs.
Don't delete any commas though, or the command will fail.